### PR TITLE
RD-537 - Add support for logoUrl and showVersionDetails parameters in theme

### PR DESCRIPTION
**Pull request by qooban**
* Added support for logoUrl and showVersionDetails theme parameters to simplify layout components usage in Stage and Composer
* Added `className` prop (with default values) to layout components to simplify its customization

### CODE REVIEW

File: src/components/layout/HeaderBanner/HeaderBanner.jsx [hunk 1]
```diff
@@ -13,27 +14,53 @@ import LicenseEdition from './LicenseEdition';
...
+
+    let url = logoUrl;
+    if (theme && theme.logoUrl) {
+        url = theme.logoUrl;
+    }
+
+    let showDetails = showVersionDetails;
+    if (theme && _.isBoolean(theme.showVersionDetails)) {
+        showDetails = theme.showVersionDetails;
+    }
```
Comment (kubama) [inline src/components/layout/HeaderBanner/HeaderBanner.jsx:46]: I'd use `{}` as default value for `theme` and then:
```suggestion
    const color = theme.headerTextColor || colors.white;
    const url = theme.logoUrl || logoUrl;
    const showDetails = _.isBoolean(theme.showVersionDetails) ?  theme.showVersionDetails : showVersionDetails;
```

File: src/components/layout/Logo/Logo.jsx [hunk 2]
```diff
@@ -1,12 +1,29 @@
...
  */
 export default function Logo({ className, style, url }) {
+    const theme = useContext(ThemeContext);
+    let logoUrl = logo;
+    if (theme && theme.logoUrl) {
+        logoUrl = theme.logoUrl;
+    }
+    if (url) {
+        logoUrl = url;
+    }
```
Comment (kubama) [inline src/components/layout/Logo/Logo.jsx:26]: This logic is not in line with what is described above

Comment (qooban) [reply src/components/layout/Logo/Logo.jsx]: Fixed

File: src/components/layout/HeaderBanner/HeaderBanner.jsx (hunk 1 above)
Comment (qooban) [reply src/components/layout/HeaderBanner/HeaderBanner.jsx]: Makes sense. Fixed.

**Review by kubama — APPROVED** [review]

**Pull request merged**